.github/workflows/blank.yml

```diff
@@ -0,0 +1,55 @@
+name: CI
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  build-docker:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Build docker image
+        run: docker build . -t vcpkg-build
+
+      - name: Save docker image
+        run: docker save -o vcpkg-build.tar vcpkg-build
+
+      - name: Upload docker image
+        uses: actions/upload-artifact@v4
+        with:
+          path: vcpkg-build.tar
+          name: vcpkg-build.tar
+          compression-level: 1
+  download:
+    needs: build-docker
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: [arm64, x64]
```

Comment:
Are there no arm64 androids?

Reply:
arm64 is listed here. Did you mean another architecture?

Reply:
Oh, I meant, "are there any non-arm64 androids?"

Reply:
No. See https://developer.android.com/ndk/guides/abis.

I'm surprised I don't see riscv64 there, but I guess it is not mature enough.

Reply:
Or if the question was about x86_64, then the desktop emulators use them.

Reply:
ok!